File: Leetcode/1051. Height Checker.rs
impl Solution {
    pub fn height_checker(heights: Vec<i32>) -> i32 {
        let mut expected = heights.to_vec();
        expected.sort_unstable();
        let mut cnt = 0;
        
        for i in 0..heights.len() {
            if heights[i] != expected[i] { cnt += 1; }
        }
        
        cnt
    }
}

File: Leetcode/TypeScript/1051. Height Checker.ts
function heightChecker(heights: number[]): number {
    let expected: number[] = [...heights];
    expected.sort((a,b) => a-b);
    let cnt: number = 0;
    for(let i=0; i<expected.length; i++) if(expected[i] != heights[i]) cnt++;
    return cnt;
};
